x-pack/plugins/viz_editor/public/pseudo_plugins/indexpattern_datasource_plugin/to_expression.ts

```diff
@@ -12,7 +12,7 @@ function isRawDocumentQuery(query: Query) {
 }
 
 function queryToEsAggsConfigs(query: Query): any {
-  return [...query.select].reverse().map((selectOperation, index) => {
+  return [...query.select].map((selectOperation, index) => {
```

Comment:
I added this because it's the way the current editor builds the aggs array - good catch on the actual ordering behavior. The shallow clone can be removed now as map doesn't mutate.

Reply:
Done